main.rb

```diff
@@ -0,0 +1,8 @@
+require_relative 'app'
+puts 'Welcome to School Library App!'
+def main
+  app = App.new
+  app.run
+end
+
+main
```

Comment:
- [ ] [OPTIONAL] Good job so far! It would be clear and proffessional if you implment `main` invoked at the end of the file, about the ruby lessons about basic sintax like loops: [THIS LESSON](https://github.com/microverseinc/curriculum-ruby/blob/main/simple-ruby/lessons/basic_syntax.md) 👍🏼 

```ruby
# main.rb
def main
  app = App.new()

  response = nil

  puts "Welcome to School Library App!\n\n"
  while response != "7"
    puts "Please choose an option by enterin a number:"
    puts "1 - List all books"
    puts "2 - List all people"
    puts "3 - Create a person"
    puts "4 - Create a book"
    puts "5 - Create a rental"
    puts "6 - List all rentals for a given person id"
    puts "7 - Exit"
    # *Lets users choose an option.
    response = gets.chomp

    case response
    when "1"
      app.list_books()
    when "2"
      app.list_people()
    when "3"
      app.create_person()
    when "4"
      app.create_book()
    when "5"
      app.create_rental()
    when "6"
      app.list_rentals_for_person_id()
    when "7"
      # *Have a way to quit the app.
      puts "Thank you for using this app!"
    end

    puts "\n"
  end
end

main()
```

Reply:
Thank you! I will research this and try to apply it!